README.md

```diff
@@ -2,6 +2,8 @@
 
 Skyscanner's stylelint config.
 
+This stylelint config is based on `stylelint-config-skyscanner` with Skyscanner specific additions and is meant to be used with Backpack and `backpack-react-scripts`. Rules encourage developers to leverage the Backpack design system and to outsource development dependencies to `backpack-react-scripts`.
```

Comment:
this config IS `stylelint-config-skyscanner` ... isn't it based on `stylelint-config-standard`?